stubs/pest-tests/EmailVerificationTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use App\Models\User;
+use App\Providers\RouteServiceProvider;
+use Illuminate\Auth\Events\Verified;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Support\Facades\URL;
+use Laravel\Fortify\Features;
+
+test('email verification screen can be rendered', function () {
+    $user = User::factory()->withPersonalTeam()->create([
+        'email_verified_at' => null,
+    ]);
+
+    $response = $this->actingAs($user)->get('/email/verify');
+
+    $response->assertStatus(200);
+})->skip(function () {
+    return ! Features::enabled(Features::emailVerification());
+}, 'Email verification not enabled.');
+
+test('email can be verified', function () {
+    Event::fake();
+
+    $user = User::factory()->create([
+        'email_verified_at' => null,
+    ]);
+
+    $verificationUrl = URL::temporarySignedRoute(
+        'verification.verify',
+        now()->addMinutes(60),
+        ['id' => $user->id, 'hash' => sha1($user->email)]
+    );
+
+    $response = $this->actingAs($user)->get($verificationUrl);
+
+    Event::assertDispatched(Verified::class);
+
+    expect($user->fresh()->hasVerifiedEmail())->toBeTrue();
+    $response->assertRedirect(RouteServiceProvider::HOME.'?verified=1');
+})->skip(function () {
+    return ! Features::enabled(Features::emailVerification());
+}, 'Email verification not enabled.');
+
+test('email can not verified with invalid hash', function () {
```

Comment:
Missing word: "email can not **be** verified with invalid hash"

(This missing word is also present in the original PHPUnit test. Might be worth correcting it at the same time?)